### PR TITLE
REGISTRY-3227 Selenium long running tests for publisher v2

### PR DESCRIPTION
**Pull request by lahirue**

### CODE REVIEW

File: modules/integration/tests-ui-integration/tests-es-ui/src/test/java/org/wso2/carbon/greg/publisher/lifecycles/CustomLifecycleUITestCase.java [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+            lifeCycleAdminServiceClient
+                    .addLifeCycle(readFile(resourcePath + "lifecycle" + File.separator + "lifecycleUI.xml"));
+        } catch (AxisFault e) {
+            log.error("Error while adding new configurations", e);
+        }
+        publisherLoginPage.loginAs(automationContext.getContextTenant().getContextUser().getUserName(),
+                automationContext.getContextTenant().getContextUser().getPassword());
+    }
+
+/*    @Test(groups = "wso2.greg", description = "Create custom RXT instance and validate its availability ")
```
Comment (sameerak) [inline modules/integration/tests-ui-integration/tests-es-ui/src/test/java/org/wso2/carbon/greg/publisher/lifecycles/CustomLifecycleUITestCase.java]: remove commented out code

File: modules/integration/tests-ui-integration/tests-es-ui/src/test/java/org/wso2/carbon/greg/publisher/lifecycles/CustomLifecycleUITestCase.java [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    @Test(groups = "wso2.greg", description = "comment and promote Lifecycle Operation by new user",
+            dependsOnMethods = "testLogAsUserWithPermission")
+    public void testCommentAndPromoteAgain() throws MalformedURLException, XPathExpressionException {
+        PublisherUtil asset = new PublisherUtil(driver);
+        driver.findElement(By.id(uiElementMapper.getElement("publisher.lifecycle"))).click();
+        driver.findElement(By.xpath(uiElementMapper.getElement("publisher.promote.checkbox1.xpath"))).click();
+        asset.addLCComment("going to promote");
+        // since permission is not working for custom RXT
+      /*  assertTrue(isElementPresent(driver, By.id(uiElementMapper.getElement("publisher.promote"))),
+                "Promote button not available in lifecycle UI for test case " + log.getClass().getName()); */
```
Comment (daneshk) [inline modules/integration/tests-ui-integration/tests-es-ui/src/test/java/org/wso2/carbon/greg/publisher/lifecycles/CustomLifecycleUITestCase.java]: remove commented out code

Comment (lahirue) [inline modules/integration/tests-ui-integration/tests-es-ui/src/test/java/org/wso2/carbon/greg/publisher/lifecycles/CustomLifecycleUITestCase.java]: fixed with this PR https://github.com/wso2/product-greg/pull/847

File: modules/integration/tests-ui-integration/tests-es-ui/src/test/java/org/wso2/carbon/greg/publisher/lifecycles/CustomLifecycleUITestCase.java (hunk 1 above)
Comment (lahirue) [inline modules/integration/tests-ui-integration/tests-es-ui/src/test/java/org/wso2/carbon/greg/publisher/lifecycles/CustomLifecycleUITestCase.java]: this commented code block has been already removed.

**Pull request merged**